packages/klevu-ui/src/components/klevu-chat-messages/klevu-chat-messages.tsx

```diff
@@ -93,7 +136,7 @@ export class KlevuChatMessages {
                     innerHTML={htmlMessage}
                   ></klevu-chat-bubble>
                   {this.enableMessageFeedback && message.message.collectFeedback && !givenFeedback && isLastMessage && (
-                    <div class="feedback">
+                    <div class={`feedback ${!this.typeWriterEnds ? 'feedback-hide' : ''}`}>
```

Comment:
can this be rendered conditionally instead of using css?

Reply:
we need to reserve space for it, otherwise, it will change width of chat bubble when effect ends